src/app/container/files/files.component.html

```diff
@@ -15,8 +15,11 @@
   -->
 
 <mat-tab-group mat-stretch-tabs>
+  <!-- Keep this as lazy loaded -->
```

Comment:
Why? (in comments)

Reply:
According to Gary this section has been flip flopping between lazy and eager, so this is just to emphasize in the future not to change it back.